packages/components/package.json

```diff
@@ -25,26 +26,28 @@
     "scripts": {
         "prebuild": "yarn clean",
         "build": "run-s build:* copy-files",
-        "build:js": "tsc --build ./tsconfig.build.json",
+        "build:js": "tsup",
+        "build:types": "tsc --build ./tsconfig.build.json",
         "copy-files": "copyfiles -u 1 \"src/**/*.css\" dist",
         "clean": "rimraf dist tsconfig.build.tsbuildinfo"
     },
     "peerDependencies": {
-        "react": "^18.0.0",
-        "react-dom": "^18.0.0",
         "@hopper-ui/styled-system": "*",
-        "react-aria-components": "*"
+        "react": "^18.0.0",
+        "react-aria-components": "*",
+        "react-dom": "^18.0.0"
     },
     "dependencies": {
         "@hopper-ui/icons": "^2.1.0",
-        "@popperjs/core": "2.11.2",
-        "react-is": "17.0.2",
-        "text-mask-core": "5.1.2",
-        "use-debounce": "7.0.1"
+        "react-aria-components": "^1.1.1",
+        "react-is": "17.0.2"
     },
     "devDependencies": {
         "@hopper-ui/styled-system": "^2.2.0",
-        "react-aria-components": "^1.1.1"
+        "@popperjs/core": "2.11.2",
```

Comment:
you moved 3 dependencies to dev dependencies, which i think will break code for consumers? use-debounce, text-mask-code and popperjs must be shipped with client code